listener/utils/fake_logger.py

```diff
@@ -0,0 +1,85 @@
+"""
+Fake logger lets you populate a dummy log file. It will be filling it with random gibberish for the
+time you specify. Once that time has passed, a target log message will be added to dummy log file.
+Target message means a line that you're looking for. This can be used for unit testing or (mainly)
+for development purposes.
+"""
+import argparse
+import pathlib
+import string
+import time
+
+
+from random import choices, randint
+
+
+""" Wishlist:
+This is lazy implementation of this utility. If we find it useful, we might want to:
+- Provide a template for fake log messages (e.g. with timestamps)
+- Let users define their own templates
+- There's probably better solution than catching KeyboardInterrupt for indefinite running
+- Display content of log file while running this script
+"""
+
+
+TARGET_MSG = "This one!"
+
+
+def generate_random_string(target_msg, max_length=80):
+    """Generate random string of letters and digits that is between 1 and max_length chars long."""
+    random_str = target_msg
+    while target_msg in random_str:  # Make sure target_msg in not included in generated string
+        random_chars = choices(string.ascii_letters + string.digits, k=randint(1, max_length))
+        random_str = ''.join(random_chars) + '\n'
+    return random_str
+
+
+def main():
+    parser = argparse.ArgumentParser(description=__doc__)
```

Comment:
Replace argparse for click. 
Example here:
https://github.com/grafuls/disruption_generator/commit/55be1a03382696bc0935cb7e09627ffa35e44294#diff-9a66eb29f78de6b36ed9340ebf6dd7f4R17